Python/sort/Bucket_Sort.py

```diff
@@ -0,0 +1,65 @@
+# Bucket Sort Algorithm
+
+# Function to perform Insertion Sort
+def insertion_sort(array, size):

```

Comment:
Placing of statements in insertion sort function is exact replica of GFG code only with minor changes in variable names.